ksonnet-gen/ksonnet/strings.go

```diff
@@ -0,0 +1,72 @@
+package ksonnet
+
+import (
+	"strings"
+)
+
+func camelCase(in string) string {
+	out := ""
+
+	for i, r := range in {
+		if i == 0 {
+			out += strings.ToLower(string(r))
+			continue
+		}
+
+		out += string(r)
+
+	}
+
+	return out
+}
+
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
+
+var (
+	capTransforms = [][]string{
```

Comment:
Instead of this array, is it worth it to write another function like `camelCase` to lower case all letters except the first?

Reply:
I pulled this in because it was in the old code. We don’t have to do this, but it is in the older code. I’ll take a look to see if I can make this work with a function instead of the static list.